docs/DeveloperGuide.md

```diff
@@ -619,6 +652,28 @@ testers are expected to do more *exploratory* testing.
 
 ---
 
+### Adding comment
+
+1. Deleting a show while all shows are being shown
+
+    1. Prerequisites: List all shows using the `list` command. Multiple shows in the list.
+
+    1. Test case: `comment 1 c/Hello`<br>
+       Expected: Comment of the first show in the list is edited to "Hello".
+
+    1. Test case: `comment 1`<br>
+       Expected: Comment of the first show in the list is deleted.
+
+    1. Test case: `comment 0`<br>
+       Expected: No comments are edited. Error details shown in the status message. Status bar remains the same.
+
+    1. Other incorrect delete commands to try: `comment`, `comment x` (where x is larger than the list size)<br>
```

Comment:
This should be comment command instead of delete commands?

Reply:
Edited according!

Was debating if this use case was necessary given the duplication of use case that was mentioned during the week 10 tutorial, considering that it is similar to edit command!

Would like to hear your input on it!